### PR TITLE
dotsies: added https to homepage

### PR DESCRIPTION
**Pull request by AlwaysLivid**

### CODE REVIEW

File: media-fonts/dotsies/dotsies-1.0~20181222.recipe [hunk 1]
```diff
@@ -4,10 +4,10 @@ by hand, they take up a lot of unnecessary space. Your eyes have to move at \
 a frantic pace from left to right to read. The letters in Dotsies words \
 smoosh together, so words look like shapes! Get more screen space! Save \
 paper. Have a more comfortable reading experience."
-HOMEPAGE="http://dotsies.org/"
+HOMEPAGE="https://dotsies.org/"
 COPYRIGHT="2012 Craig Muth"
 LICENSE="CC-BY-4.0"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="http://dotsies.org/Dotsies.ttf#noarchive"
```
Comment (korli) [inline media-fonts/dotsies/dotsies-1.0~20181222.recipe]: What about SOURCE_URI?

Comment (AlwaysLivid) [reply media-fonts/dotsies/dotsies-1.0~20181222.recipe]: That one still slips away, sorry about that.

**Comment by Begasus** [comment]
Why closed this? Only needed https for SOURCE_URI as @korli suggested?

**Comment by AlwaysLivid** [comment]
> Why closed this? Only needed https for SOURCE_URI as @korli suggested?

I mistakenly deleted the repository while cleaning up my account while I had forgotten that there were astounding pull requests. I'm going to try to handle it.

**Pull request closed without merging**